src/main/java/mate/academy/springboot/datajpa/model/Product.java

```diff
@@ -0,0 +1,61 @@
+package mate.academy.springboot.datajpa.model;
+
+import java.math.BigDecimal;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "products")
+public class Product {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private BigDecimal price;
+    @OneToOne
+    private Category category;
```

Comment:
how many products can belong to one category?

Reply:
in README is written that one product can have one category...